com.usebottles.bottles.yml

```diff
@@ -0,0 +1,69 @@
+---
+app-id: com.usebottles.bottles
+runtime: org.gnome.Sdk
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+add-extensions:
+  org.gnome.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '3.38'
+command: bottles
+finish-args:
+- --allow=multiarch
+- --share=network
+- --device=dri
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --socket=pulseaudio
+- --socket=session-bus
+- --filesystem=~/Games
+- --filesystem=~/wineprefixes
```

Comment:
Similar question as with XDG_DATA_HOME.

Reply:
These are paths used by other wineprefix managers. We must be able to read them in order to import wineprefix from a manager external to ours. Maybe there is a better way to declare these paths?

Reply:
Is it just read only? If so, add :ro afterwards.